content/docs/tasks/security/vault-ca/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Istio Vault CA integration
+description: Tutorial on how to plug in a Vault CA for issuing certificates in Istio.
+weight: 10
+keywords: [security,certificate]
+---
+
+This tutorial walks you through an example to plug in a Vault CA for issuing
+certificates in Istio.
+
+## Before you begin
+
+* Create a new Kubernetes cluster to run the example in this tutorial.
+
+* On GKE, you may set the value of the `CLUSTER`, `ZONE`, and `PROJECT`
```

Comment:
Do we need per-platform instructions? It's not great to have an explicit callout to GKE in an otherwise generic doc page.

Reply:
Removed the per-platform instructions.